src/main/java/twilightforest/TFTickHandler.java

```diff
@@ -40,7 +33,11 @@
  */
 public class TFTickHandler {
 
-    public Item portalItem = null;
+    private final Item portalItem;
```

Comment:
Does this work in full pack? Probably no one is writing to this field, but just in case.

Reply:
I checked that the public fields that I deleted aren't used anywhere else with the classdump of the pack in recaf. Also I tested it in full pack and it works properly